db/migrate/20241113185651_add_external_billing_id_on_enterprises.rb

```diff
@@ -0,0 +1,5 @@
+class AddExternalBillingIdOnEnterprises < ActiveRecord::Migration[7.0]
+  def change
+    add_column :enterprises, :external_billing_id, :string, limit: 128
```

Comment:
I put the limit at 128 chars as it seems enough, but it can be of course changed in case.